storybook/stories/core/components/Input/SelectInputButton.stories.js

```diff
@@ -132,13 +132,15 @@ Default.args = {
   button: buttonObject,
   moreIconName: 'oxd-info',
   doubleLineLabel: true,
+  additionalIconName: 'oxd-best-match'
```

Comment:
no need to add it to every story. just create a new story where additional icon is visible

Reply:
Done. A new story is created